public/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>TimeStamp</title>
+    <link rel="stylesheet" href="/css/main.css" />
+  </head>
+  <body>
+    <div class="center"><h1>TimeStamp Microservice</h1></div>
+
+</br>
+</br>
+<div class="center">
+<h1>Example Usage:</h1>
+</div>
+</br>
+
+<div class="center">
+    <h3>Natural Language:</h3>
+</div>
+
+<div class="center">
+        <p class="highlight">http://time.jma.cloud:3000/feb%2025,%202017 </p>
```

Comment:
And also, can you fix the formatting on this?

Reply:
I'm not sure what you mean?

Reply:
I updated html format not sure if that what your wanting

Reply:
<img width="677" alt="screen shot 2018-12-10 at 4 07 49 pm" src="https://user-images.githubusercontent.com/3806031/49769654-dcd3e100-fc95-11e8-9402-f140383b467d.png">


I'm not sure what indententation you're using but try to be consistent. So here I was pointing out the large space. It should be 2 or 4 spaces. It looks like it's more than that but my eyes could be deceiving me.